app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  belongs_to :restaurant
+  belongs_to :order
+  validates :name, presence: true,
+  validates :email, presence: true, format: { with: /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i }
```

Comment:
Should we consider email to have uniqueness: true, or are we OK to have different users sharing the same email id? In which case, it may be a bit confusing later as to which named user actually ordered something

Reply:
```suggestion
  validates :email, presence: true, format: { with: /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i }
```
Agree with @saikatb221062, there should be uniqueness: true when validating email addy. Regarding the format, we can remove it and let simple_form gem handle format validation.